docs/usage/writing-delta-tables.md

```diff
@@ -50,4 +50,17 @@ that partition or else the method will raise an error.
 ```
 
 This method could also be used to insert a new partition if one doesn't
-already exist, making this operation idempotent.
\ No newline at end of file
+already exist, making this operation idempotent.
+
+## Overwriting part of the table data with `replaceWhere`
```

Comment:
Using the replaceWhere naming is a bit odd, it's not spark-delta and our api interface uses predicate solely (or the partition_filter which we will eventually phase out).

Reply:
Agree, it's not ideal because it doesn't match the Python's parameter name, more like a concept, happy to change if you have a better name in mind

Reply:
This fits better:

Overwriting part of the table data with a SQL `predicate`

Reply:
@MrPowers - your take on this?